### ISSUE TITLE
Cannot Find Rubber Trees

### ISSUE THREAD

**Issue by mgwerner** [TechReborn/TechReborn#1152]
MC 1.11.2, Forge 13.20.0.2300 TR 2.3.3.174

I have scoured the map, spending hours wandering through forest and swamplands, but no rubber trees.  Have been to 10-15 villages and ruins, no sapling in chests.  Not using IC2 with TR, only TR as my tech mod.  Anyone run into this?

**Comment by modmuss50** [comment]
They are there, just very rare for some reason. I have made them a bit more common by default

**Comment by mgwerner** [comment]
Is there a way to increase the rate?  I started a new world in case something was wrong, still found none.

**Comment by modmuss50** [comment]
After playing around with it, they dont seem to be spawning at all. The odd one might work.

**Comment by mgwerner** [comment]
Well, that pretty much breaks things.  Was it working in a prior build, like 167?  I can drop back to that and delete the regions for now.

**Comment by mgwerner** [comment]
I have tried different versions, down to 142, made several creative worlds, and flown around searching.  No rubber trees that I can find in the numerous forests and swamps.  I never found a jungle, but in the hours I have spent looking I should have found a clump, or at least one, right?  I loaded over 100 regions each time searching.

Disappointingly, I will have to add back in IC2 EXP, since the lack of rubber is a serious break in the progression.
